README.md

```diff
@@ -72,20 +72,14 @@ restrictions.
 Let's start with the simple example of posting "Hello World" on the Android
 device's screen. Create the following files:
 
-**sample_config.json**
-
-```python
-{
-    "testbed":
-    [
-        {
-            "_description": "A testbed where adb will find Android devices.",
-            "name": "SampleTestBed",
-            "AndroidDevice": "*"
-        }
-    ],
-    "logpath": "/tmp/logs"
-}
+**sample_config.yml**
+
+```yaml
+TestBeds:
```

Comment:
Are we going with testbed as one word or two? If one perhaps we should lowercase all the B's in labels and names (for consistency)